design/features/payment-links/version-1.md

```diff
@@ -130,98 +134,118 @@ The page ends with a text input labelled “Enter the URL of your GOV.UK Pay pay
 
-#### Add a link error summary
+#### “Add a link” error summary
 
 ![“There is a problem” error summary notification. Screenshot](./screenshots-v1/002-add-a-link-to-a-payment-page-on-govuk-pay-error-summary.png)
 *The error message that is linked to the input says “Enter a link in the correct format, like https​://www.gov.uk/payments/organisation/service”*  
 
-### Create a form - payment link added
+### “Create a form” - payment link added
 
 ![Create a form task list page showing green “Your payment link has been saved” success notification at the top. Screenshot](./screenshots-v1/003-create-a-form-added-payment-link.png)
-*The “Optional tasks” summary list item, “Add a link to a payment page on GOV.UK Pay”, now shows ‘Completed’ with no background to the right side of the row.*
+*The “Optional tasks” summary list item, “Add a link to a payment page on GOV.UK Pay”, now shows as ‘Completed’ (with no background shading) to the right side of the row.*
 
-#### Payment link successfully added notification
+#### “Payment link successfully added” notification
 
 ![Green “Success” notification box. Screenshot](./screenshots-v1/003-success-notification.png)
-*The message says “Your payment link has been saved”* 
+*The message says “Your payment link has been saved”.* 
 
 
-### Create a form - landing on the task list page that already has a payment link  
+### “Create a form” - landing on the task-list page that already has a payment link  
 
-![Create a form task list page showing “Add a link to a payment page on GOV.UK Pay” marked as ‘completed’. Screenshot](./screenshots-v1/004-create-a-form-return.png)
+![“Create a form” task-list page showing “Add a link to a payment page on GOV.UK Pay” marked as ‘completed’. Screenshot](./screenshots-v1/004-create-a-form-return.png)
 
-When the form creator returns to the “Create a form” task list, if there is already a payment link added then this row appears with ‘Completed’ on the right side of the row. There is still no increase in number of tasks completed.  
+When the form creator returns to the “Create a form” task list, if there’s already a payment link added then this row appears with ‘Completed’ on the right side of the row. There’s still no increase in the number of tasks showing as completed.  
 
 ___
 
-## Preveiwing the form fillers journey
+## Previewing the form filler’s journey
 
 ### Confirmation screen preview 
 
 ![“You still need to pay” heading inside a blue box replacing the usual green confirmation box. Screenshot](./screenshots-v1/1000-preview-confirmation-still-need-to-pay.png)  
 
-This is an alternative confirmation screen used where a payment link has been added to a form. It shows a blue background panel with the heading “You still need to pay” followed by “Your form reference number is:” and the unique generated reference number for this forms submission.  
```

Comment:
@C-Harry why was this removed? 
I feel like this context is missing (especially as a descriptive for anyone who struggles to see the images)

Reply:
This must have been an accident @christophercameron-ixd - I'm still pretty clunky with the Mac and occasionally use the Windows shortcuts (I'm on a Windows keyboard). So, in short, this must've been an error - apologies!!

Reply:
The only thing I'd have meant to change was adding an additional apostrophe to "forms submission" - so it would be "form's submission"